package.json

```diff
@@ -57,6 +57,7 @@
     "rollup-plugin-minify-html-literals": "^1.2.6",
     "rollup-plugin-terser": "^7.0.2",
     "serve": "^13.0.2",
+    "serve-handler": "^6.1.3",
```

Comment:
We would've found this failing when we removed the last package that has `serve-handler` as a dependency.

Reply:
I don't even remember what it does...

Reply:
It's used in dev-server